examples/demo/src/main/scala/Main.scala

```diff
@@ -43,6 +45,53 @@ object Main {
       DataRow().addCell("school", "Well Known School").addCell("numStudents", 30000)
     )
 
+    val recordHeaders: List[DataRecord] = List(
+      DataRecord()
+        .addCell("dateHeader", "Date")
+        .addRecord(List(
+          DataCell("peerCloseHeader", "AAPL close"),
+          DataCell("peerChangeHeader", "AAPL")
+          ))
```

Comment:
This could just be addCells. Either way we need to move away from copy entire template row, look at the key in each cell and replace it with data matching the key _to_ copying the cells in each row for each record.